packages/request-manager/src/events/bootstrap.ts

```diff
@@ -6,7 +6,11 @@ const clientStatusGroupParser = (message: string): RegExpMatchArray | null =>
     message.trim().match(/^(^650 STATUS_CLIENT NOTICE BOOTSTRAP PROGRESS=.*)+/gm);
 
 const statusParser = (status: string): RegExpMatchArray | null =>
-    status.trim().match(/^650 STATUS_CLIENT NOTICE BOOTSTRAP (PROGRESS=(?<progress>\d+)?.*)?/);
+    status
+        .trim()
+        .match(
+            /^650 STATUS_CLIENT NOTICE BOOTSTRAP (PROGRESS=(?<progress>\d+)?)? TAG=(?:[\w\s]+) (SUMMARY="(?<summary>.+)")?/,
+        );
```

Comment:
spec says there might also be "NOTICE BOOTSTRAP" or "WARN BOOTSTRAP".  https://gitweb.torproject.org/torspec.git/tree/control-spec.txt#n982 Do you have any idea when the latter could happen?

Reply:
To be honest I am not totally sure when that event will be triggered, I have not seen it yet, but we could investigate a bit to get some more inputs from there.

Reply:
thats it. again, I think it might be useful to look for this event as well and just log it or somethig. this can be done anytime later. feel free to resolve this. I am just trying to look behind the corner a bit